docs/tutorials/platform_beginner/6_enabling_account_registration.md

```diff
@@ -0,0 +1,71 @@
+```markdown
+
+```
+
+# Step 6 - Enabling account registration
+
+We are trying to let our friends write their bike trips directly into our website. We would like to let them create Rides. As an Administrator of the website you could create the Users yourself but In order to let them create their own content, first, you want to let them create their own accounts.
+
+## Add the registration possible
+
+From the Admin Panel, go to the _Roles_ management screen, click on _Anonymous_ Role name link to edit it.
```

Comment:
`From the Admin Panel, go to the _Roles_ management screen and click on the _Anonymous_ Role name to edit it.`